docs/gcal_workweek_sample.json

```diff
@@ -0,0 +1,137 @@
+[
```

Comment:
I should've raised this sooner: my intention was that the docs/ directory had just the verbatim output from the output. We'd pass the API output around, document them in the docs directory and then construct the document to store in firebase from those documents. 

Right now, in places (e.g. [`fs.py`][https://github.com/mozilla-mobile/prox-server/blob/master/app/providers/fs.py], we're constructing transitionary documents to pass around, and then documenting these transitionary docs here.